cfgov/jinja2/v1/budget/strategic-plan/index.html

```diff
@@ -12,16 +12,17 @@
 {%- endblock %}
 
 {% block content_main %}
-
-    <h1>
-        Strategic Plan
-    </h1>
-    <p class="h3">
-        The CFPB Strategic Plan outlines four goals in pursuit of the Bureau’s mission.
-        For each, the plan also describes desired outcomes, required strategies and investments,
-        and specific measures of success. This is the work you should expect us to take
-        on over the coming years.
-    <p>
+    <div class="block block__flush-top">
+        <h1>
+            Strategic Plan
+        </h1>
+        <p class="h3">
```

Comment:
If this is an intro paragraph, it should be `lead-paragraph` now instead of an `h3` class.